docs/Node Operators/best-practices-aws.md

```diff
@@ -0,0 +1,223 @@
+        "output": ""
+      }
+    },
+
+    ⋮
+
+  ]
+}
+```
+
+To check the status of your Chainlink node container, use the `docker ps` command.
+
+
+## Recovering or Upgrading your Chainlink node container
+
+Tag versions for Chainlink node releases are available in the [Chainlink docker hub](https://hub.docker.com/r/smartcontract/chainlink/tags)
+
+If the Chainlink node containers are stopped or if you need to start up a new instance during an upgrade or recovery, you must create new `.env`, `.password`, and `.api` files to start the Chainlink node.
+
+**Generating the .env file:**
+
+```
+cd /home/ec2-user/.chainlink/ && ./create-env.sh \
+${chainNetwork} \
+${blockchainNodeUrl} \
+${psqlUser} \
+$(aws secretsmanager get-secret-value --secret-id DBSecret --query "SecretString" --output text) \
+${psqlHostname} \
+${psqlPort} \
+${psqlDb}
+```
+
+**Generating the .password file:**
+
+```
+cd /home/ec2-user/.chainlink/ && ./create-password.sh \
+$(aws secretsmanager get-secret-value --secret-id WalletSecret --query "SecretString" --output text)
+```
+
+**Generating the .api file:**
+
+```
+cd /home/ec2-user/.chainlink/ && ./create-api.sh \
+${apiUser} \
+$(aws secretsmanager get-secret-value --secret-id ApiSecret --query "SecretString" --output text)
+```
+
+**Stopping and removing the existing Chainlink node container:**
+
+This process is required when you update the container to a newer release.
+
+```
+docker stop chainlink && docker rm chainlink
+```
+
+**Starting the Chainlink node container in Docker:**
+
+```
+latestimage=$(curl -s -S "https://registry.hub.docker.com/v2/repositories/smartcontract/chainlink/tags/" | jq -r '."results"[]["name"]' | head -n 1)
+cd /home/ec2-user/.chainlink && docker run -d \
+--log-driver=awslogs \
+--log-opt awslogs-group=ChainlinkLogs \
+--restart unless-stopped \
+--name chainlink \
+-p 6688:6688 \
+-v /home/ec2-user/.chainlink:/chainlink \
+--env-file=/home/ec2-user/.chainlink/.env  smartcontract/chainlink:$latestimage local n \
+-p /chainlink/.password
+-a /chainlink/.api
+```
+
+You can find more details with maintenance and examples in the [Performing System Maintenance](https://docs.chain.link/docs/performing-system-maintenance/) guide.
+
+Additionally, follow the directions in the [User Guide for Linux Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-updates.html) to keep your Amazon Linux instance software up to date.
+
+
+## Backup and Restore
+
+### Database Backup
+
+The PostgreSQL database is critical because it stores most of the Chainlink node data. Aurora automatically backs up your PostgreSQL cluster during the backup window with a default backup retention of 7 days. You can optionally modify the database cluster’s backup retention period from to a maximum of 35 days. For more details, see the guide for [backing up and restoring an Aurora cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Backups.html).
```

Comment:
Missing the from <minimum_amount_of_days>:

You can optionally modify the database cluster’s backup retention period from 1 day to a maximum of 35 days. For more details, see the guide for [backing up and restoring an Aurora cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Backups.html).

Reply:
Good catch. Change applied.